ocs_ci/ocs/ui/base_ui.py

```diff
@@ -109,6 +132,23 @@ def take_screenshot(self):
         logger.info(f"Creating snapshot: {filename}")
         self.driver.save_screenshot(filename)
 
+    def get_text(self, locator, timeout=30, expected_text=None):
```

Comment:
The method name `get_text` is slightly misleading here - it sounds like it will return you some text from element, but it only compare it with some expected string.

Also it looks like the `expected_text` parameter should be mandatory (it doesn't make much sense to omit it when calling this method), but the `timeout` parameter is clearly optional. So it will probably be better to put the `expected_text` parameter before `timeout` parameter and do not set any default value for the `expected_text` parameter.